src/web/src/app/space/list/component.ts

```diff
@@ -0,0 +1,322 @@
+	];
+
+	links: SpaceLink[] = [
+		{id: crypto.randomUUID(), spaceId: '11111111-1111-1111-1111-111111111111', creatorId: "vasya", name: "link1", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '22222222-2222-2222-2222-222222222222', creatorId: "vasya", name: "link2", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '33333333-3333-3333-3333-333333333333', creatorId: "vasya", name: "link3", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '44444444-4444-4444-4444-444444444444', creatorId: "vasya", name: "link4", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '55555555-5555-5555-5555-555555555555', creatorId: "vasya", name: "link5", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '66666666-6666-6666-6666-666666666666', creatorId: "vasya", name: "link6", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '77777777-7777-7777-7777-777777777777', creatorId: "vasya", name: "link7", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+		{id: crypto.randomUUID(), spaceId: '88888888-8888-8888-8888-888888888888', creatorId: "vasya", name: "link8", createdAt: new Date("2023-10-08"), expiredAt: new Date("2024-10-08")},
+	];
+
+	receivedInvites: SpaceInvitation[] = [];
+	receivedLinks: SpaceLink[] = [];
+	receivedSpaces: Space[] = [];
+
+	@ViewChild('invitesPaginator') invitesPaginator: MatPaginator;
+	@ViewChild('linksPaginator') linksPaginator: MatPaginator;
+	@ViewChild('spacesPaginator') spacesPaginator: MatPaginator;
+	@ViewChild('scrollToInviteForm') scrollToInviteForm: ElementRef;
+	@ViewChild('scrollToLinkForm') scrollToLinkForm: ElementRef;
+
+	constructor(
+		@Inject(DOCUMENT) private document: any,
+		private fb: FormBuilder,
+	) {
+		this._initInviteForm();
+		this._initLinkForm();
+	}
+
+	ngOnInit() {
+		this.getInvites(this.invitesLowValue, this.invitesHighValue);
+		this.getLinks(this.linksLowValue, this.linksHighValue);
+		this.getSpaces(this.spacesLowValue, this.spacesHighValue);
+	}
+
+	getInvites(lowValue, highValue) {
+		this.receivedInvites = this.invites.slice(lowValue, highValue);
+	}
+
+	getPaginatorInvites(event) {
+		if(event.pageIndex === this.invitesPageIndex + 1) {
+			this.invitesLowValue = this.invitesLowValue + this.invitesPageSize;
+		   	this.invitesHighValue =  this.invitesHighValue + this.invitesPageSize;
+		} else if(event.pageIndex === this.invitesPageIndex - 1) {
+			this.invitesLowValue= this.invitesLowValue - this.invitesPageSize;
+			this.invitesHighValue =  this.invitesHighValue - this.invitesPageSize;
+		}   
+		this.invitesPageIndex = event.pageIndex;
+		this.getInvites(this.invitesLowValue, this.invitesHighValue);
+	}
```

Comment:
Тут space это SpaceService
Пример getInvitationList в комментарии выше, только там getSpaceList

```suggestion
	getInvites(limit: number, page: number) {
		this.space.getInvitationList(limit, page)
			.subscribe(res => {
				this.invitesPaginator.length = res.count;
				this.invites = res.results;
			});
	}
```

Reply:
Сделал. Только, как писал выше, в пагинатор на стороне компонента length не получилось передать, поэтому стал использовать переменную. Планирую вернуться к этому (к length) завтра - разобраться.

Reply:
Ответил выше про недоступность пагинатора сразу после инициализации. Примерно советуют ещё вот такое https://stackoverflow.com/a/59445975 , но мне не до конца понятно, какие это может эффекты принести, поэтому не использую пока. Но если точно нужно нам это, то скажи, займусь.